test/extended/util/nodes.go

```diff
@@ -0,0 +1,51 @@
+package util
+
+import (
+	"strings"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+// GetClusterNodesBy returns the cluster nodes by role
```

Comment:
```suggestion
// GetClusterNodesByRole returns the cluster nodes by role
```

sorry this needed to change too to stay aligned.